src/ripple/protocol/impl/STParsedJSON.cpp

```diff
@@ -192,8 +202,7 @@ static boost::optional<detail::STVar> parseLeaf (
                 }
 
                 ret = detail::make_stvar <STUInt8> (field,
```

Comment:
We've already checked the range above. We can replace this with a straight `static_cast<unsigned char>` without any problems.